cmd/statusd/utils.go

```diff
@@ -1488,3 +1492,25 @@ func testValidateNodeConfig(t *testing.T, config string, fn func(common.APIDetai
 
 	fn(resp)
 }
+
+func testProfiling(t *testing.T) bool {
+	if err := common.StartCPUProfile(""); err != nil {
+		t.Error("Cannot start profiling:", err)
+		return false
+	}
+
+	address, _, _, err := statusAPI.CreateAccount(TestConfig.Account1.Password)
+	if err != nil {
+		t.Errorf("could not create account: %v", err)
+		return false
+	}
+
+	t.Log("Account created: ", address)
```

Comment:
I'd prefer not to see anything unrelated to profiling here. If you're just doing something time-consuming, it's better be something custom and provided with an appropriate comment why you're doing it.